project-rome-docs/objectivec-api/remotesystems.commanding/MCDRemoteLauncher.md

```diff
@@ -0,0 +1,46 @@
+---
+title: MCDRemoteLauncher
+description: A class used to launch an app on a remote device using a URI.
+keywords: microsoft, windows, iOS, iPhone, objectiveC, connected devices, Project Rome
+---
+
+# class `MCDRemoteLauncher` 
+
+```
+@interface MCDRemoteLauncher : NSObject
+```  
+
+A class used to launch an app on a remote device using a URI.
+
+
+## Methods
+
+### launchUriAsync
+```
+- (void)launchUriAsync:(nonnull NSString*)uri
+ withConnectionRequest:(nonnull MCDRemoteSystemConnectionRequest*)connection
+            completion:(nullable void (^)(MCDRemoteLaunchUriStatus result, NSError* _Nullable error))completionBlock;
+```
+
+Launches a URI against the Remote System specified in an [MCDRemoteSystemConnectionRequest](MCDRemoteSystemConnectionRequest.md).
+
+#### Parameters
+* `uri` The URI which will cause the launching of an app, according to its scheme.
```

Comment:
>according to its scheme [](start = 58, length = 23)

Maybe, maybe not. If the target is Windows, the target app will be chosen based on scheme. If the target is non-Windows, the target app will be chosen based on the ConnectionRequest.